packages/terra-toolkit-docs/src/terra-dev-site/tool/webpackConfigTerra/UpgradeGuide.y.tool.mdx

```diff
@@ -4,6 +4,16 @@ import { Badge } from '@cerner/webpack-config-terra/package.json?dev-site-packag
 
 # webpack-config-terra Upgrade Guide
 
+## Changes from @cerner/webpack-config-terra 1.0.0 to @cerner/webpack-config-terra 2.0.0
```

Comment:
Let's make this it's own document.

Reply:
Updated here https://github.com/cerner/terra-toolkit/runs/3116641186